### PR TITLE
Adding IP address to profile page.

### LINKED ISSUE #33: Show user IP address on profiles
Perhaps only for admins, but can be useful for connecting to gameservers

### PR DESCRIPTION
**Pull request by DinCahill**
Fixes #33

Only shows up for admins.

### CODE REVIEW

File: app/views/users/partials/ip.blade.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@if(Auth::user()->isAdmin())
```
Comment (cruelviper) [inline app/views/users/partials/ip.blade.php]: To stop the 500 error, when opening the Profil page without Login use this Code:

```
@if (Auth::check())
	@if(Auth::user()->isAdmin())
		IP: {{{ $user->ip }}}
	@endif
@endif
```

Comment (cruelviper) [inline app/views/users/partials/ip.blade.php]: To stop the 500 error, when opening the Profil page without Login use this Code:

```
@if (Auth::check())
	@if(Auth::user()->isAdmin())
		IP: {{{ $user->ip }}}
	@endif
@endif
```

Comment (cruelviper) [inline app/views/users/partials/ip.blade.php]: To stop the 500 error, when opening the Profil page without Login use this Code:

```
@if (Auth::check())
	@if(Auth::user()->isAdmin())
		IP: {{{ $user->ip }}}
	@endif
@endif
```

**Comment by cruelviper** [comment]
To stop the 500 error, when opening the Profil page without Login use this Code:

```
@if (Auth::check())
	@if(Auth::user()->isAdmin())
		IP: {{{ $user->ip }}}
	@endif
@endif
```

File: app/views/users/partials/ip.blade.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+	<small>IP: {{{ $user->ip }}}</small>
+@endif
```
Comment (cruelviper) [inline app/views/users/partials/ip.blade.php]: To stop the 500 error, when opening the Profil page without Login use this Code:

```
@if (Auth::check())
	@if(Auth::user()->isAdmin())
		IP: {{{ $user->ip }}}
	@endif
@endif
```

**Comment by ilumos** [comment]
3 years 2 days later and it's merged :)

**Pull request merged**